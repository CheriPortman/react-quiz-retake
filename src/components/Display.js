//takes headerText
//takes footerText
//takes image prop

import React from 'react';
import PropTypes from 'prop-types';

function Display({ headerText, image, footerText }) {
  
  return (
    <div>
      <p>{headerText}</p>
      <img src={image} />
      <p>{footerText}</p>
    </div>
  );
}

Display.propTypes = {
  headerText: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired,
  footerText: PropTypes.string.isRequired
};

export default Display;
