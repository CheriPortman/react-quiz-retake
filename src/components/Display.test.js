import React from 'react';
import { shallow } from 'enzyme';
import Display from './Display';

describe('Display component', () => {
  it('renders the display', () => {
    const wrapper = shallow(<Display headerText="hey" image="image.jpg" footerText="hey" />);
    expect(wrapper).toMatchSnapshot();
  });
});
