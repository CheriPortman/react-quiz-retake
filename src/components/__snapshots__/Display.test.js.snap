// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display component renders the display 1`] = `
<div>
  <p>
    hey
  </p>
  <img
    src="image.jpg"
  />
  <p>
    hey
  </p>
</div>
`;
